kong/plugins/acme/schema.lua

```diff
@@ -71,6 +71,11 @@ local schema = {
           encrypted = true, -- Kong Enterprise-exclusive feature, does nothing in Kong CE
           referenceable = true,
         }, },
+        { account_key = {
+          type = "string",
```

Comment:
we should check the validness of the key, the same check in https://github.com/Kong/kong/blob/master/kong/db/schema/entities/certificates.lua#L19 is good reference.
besides, it's a good idea to verify it's a private key instead of a public key.

Reply:
Thanks for your feedback. I've pushed the changes and added some new tests.